stack.yaml

```diff
@@ -1,7 +1,5 @@
-resolver: lts-6.25
+resolver: lts-8.0
```

Comment:
The newest snapshot is 8.5 at this point, might as well use the newest versions? https://www.stackage.org/lts-8.5